tools/isomorphic-ws.js

```diff
@@ -0,0 +1,16 @@
+var ws = null;
+
+if (typeof WebSocket !== 'undefined') {
```

Comment:
Isomorphic lib should automatically switch to ws in case of node.js and to global.WebSocket for browsers. Why we need to add this logic explicitly here.

Reply:
it is just copy of Isomorphic lib. but I've just added this line `module.exports.WebSocket = ws;`
because Isomorphic for nodejs and for webrowser has different external interface
for browser 
`import {WebSocket} from 'Isomorphic'`
for nodejs
`import WebSocket from 'Isomorphic'`